assignment/scripts/function-practice.js

```diff
@@ -49,41 +53,107 @@ console.log( 'isPositive - should say false', isPositive(-3) );
 
 // 6. Function to return the _last_ item in an array. If the 
 //    array is empty, return `undefined`.
-function getLast( array ) {
-
+function getLast(array) {
+  let lastItem = undefined;
+  if (array.length) {
+    lastItem = array[array.length - 1];
```

Comment:
Since the default value for an array lookup is `undefined`, this function can also be simplified:

```js
function getLast(array) {
  return array[array.length - 1];
}
```